ADOPTERS.md

```diff
@@ -22,6 +22,7 @@
 | Vendor | EQUINIX | | [link](https://metal.equinix.com/) | |
 | Vendor | H3C | 2019 | [link](https://www.h3c.com/en/Products_Technology/Enterprise_Products/Cloud_Computing/Cloud_Computing_Products/H3C_CloudOS/H3C_CloudOS_full-stack/) | We distribute KubeVirt as part of CloudOS to enable VM workloads on Kubernetes at customer sites. |
 | Vendor | KUBERMATIC | 2019 | [link](https://www.kubermatic.com/products/kubevirt/) | As a distributor we are running KubeVirt to enable VM workload on Kubermatic Virtualization. |
+| Vendor | KUBESPHERE | 2020 | [link](https://kubesphere.com.cn/ksv/) | KubeSphere Virtualization(KSV) provides the lightweight VM management capability based on KubeVirt. |
```

Comment:
```suggestion
| Vendor | KUBESPHERE | 2020 | [link](https://kubesphere.com) | KubeSphere Virtualization (KSV) provides lightweight VM management capability based on KubeVirt. |
```

Reply:
thanks @aburdenthehand for your quick response, We don't own that .com domain as our main site, so could we use https://kubesphere.cloud/en/ksv/ as an alternative?

Reply:
Ah, I see. Yes, that makes sense. Thanks!